third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -351,7 +352,7 @@ std::vector<ComputationClient::ComputationPtr> PjRtComputationClient::Compile(
       // to expose the knob for future reference. We can override the compiler's
```

Comment:
maybe you should update the comment here as well.

Reply:
> Can you briefly describe the code being removed?

Updated the PR description, thanks :)